tmva/tmva/inc/TMVA/DNN/Architectures/Cuda/CudaMatrix.h

```diff
@@ -151,6 +151,7 @@ class TCudaMatrix
    size_t GetNrows() const {return fNRows;}
    size_t GetNcols() const {return fNCols;}
    size_t GetNoElements() const {return fNRows * fNCols;}
+   size_t GetNElements() const {return fNRows * fNCols;}
```

Comment:
Actually It would be better to keep GetNoElements in CudaMatrix and change in CpuMatrix. This however could be another PR

Reply:
Indeed I remember our discussion, I just didn't want to further complicate the present PR by changing such a global module (`GetNelements()` is used in a lot of files, such as `MethodDL`).

I will submit the fix soon after merging this one